src/hub_predtimechart/hub_config.py

```diff
@@ -112,11 +118,73 @@ def model_output_file_for_ref_date(self, model_id: str, reference_date: str) ->
         return None
 
 
-def _validate_predtimechart_config(ptc_config: dict):
+def _validate_hub_ptc_compatibility(ptc_config: dict, tasks: dict, model_metadata_schema: dict):
     """
-    Validates `ptc_config` against the schema in schema.py.
+    Validates a hub's predtimechart compatibility as identified in README.MD > Assumptions/limitations .
 
     :param ptc_config: dict loaded from a 'predtimechart-config.yml' file
+    :param tasks: dict loaded from a hub's 'tasks.json' file
+    :param model_metadata_schema: dict loaded from a hub's 'model-metadata-schema.json' file
+    :raises ValidationError: if `tasks` is incompatible with predtimechart
+    """
+    # get the round and model_task
+    the_round = tasks['rounds'][ptc_config['rounds_idx']]
+    the_model_task = the_round['model_tasks'][ptc_config['model_tasks_idx']]
+
+    # validate: only `model_tasks` groups with `quantile` output_types will be considered
+    output_type = the_model_task['output_type']
+    if 'quantile' not in output_type:
+        raise ValidationError(f"no quantile output_type found. found types: {list(output_type.keys())}")
```

Comment:
If there are multiple validation errors (e.g., missing quantile levels and not a designated model), it would be user-friendly to return all of them at once (instead of raising an error on the first failure).